README.md

```diff
@@ -11,51 +11,28 @@ The current stage of the software is *Alpha*.
 
 So far, Open-SIR provides an implementation of the SIR model and the novel [SIR-X model, developed by Maier and Dirk](https://science.sciencemag.org/content/early/2020/04/07/science.abb4557.full) from the [Robert Koch Institut](http://rocs.hu-berlin.de/corona/docs/forecast/model/#sir-x-dynamics-outbreaks-with-temporally-increasing-interventions).
 
-## Development Setup
-
-Open-SIR uses [Pipenv](https://pipenv.pypa.io/en/latest/) to automatically create a virtual environment and manage python packages. The python packages required by Open-SIR are listed in the [Pipfile](Pipfile).
-
 ### Dependencies
 
-* Python 3.7
-* Pipenv
-
-### Installation
-
-After cloning the repository, change the current directory to the repository via `cd open-sir` and automatically install the environment from the Pipfile using Pipenv:
-```
-pipenv install
-```
-Next, activate the Pipenv shell:
-```
-pipenv shell
-```
-You can run the following command to check that the installation succeeded.
-```
-pipenv run start -p '[0.95,0.38]' -i '[341555,445,0]' -t 6
-```
-
+* Python >= 2.7
```

Comment:
Python 2.7 is deprecated and I think we should not promote its usage. If we are going to support python versions over a certain number, I think they should be at least 3.x

What do you think?

Reply:
you are right. Let's remove Python 2.7 from there.
Python >= 3.2 should be ok

Reply:
Cool 💪